rspotify-http/Cargo.toml

```diff
@@ -14,6 +14,8 @@ keywords = ["spotify", "api"]
 edition = "2018"
 
 [dependencies]
+rspotify-model = { path = "../rspotify-model", version = "0.10.0" }
```

Comment:
Is it necessary to include `rspotify-model`?

Reply:
Yes, because it's used in examples in the documentation